src/modules/asab-config/ConfigContainer.js

```diff
@@ -33,13 +33,23 @@ export default function ConfigContainer(props) {
 					/>
 				</Col>
 				<Col md={{ size: 6, offset: 1 }}>
-					<ConfigEditor
-						app={App}
-						configType={configType}
-						configName={configName}
-					/>
+					{configType != 'type' && configName != 'name' ?
+						<ConfigEditor
+							app={App}
+							configType={configType}
+							configName={configName}
+						/>
+					:
+						<React.Fragment>
```

Comment:
This is called "empty state" -> https://www.toptal.com/designers/ux/empty-state-ux-design

Very interesting topic, btw.


Also, you don't need to use "React.Fragment" when you have only one element in that.